diff --git a/node_modules/@marko/build/dist/index.js b/node_modules/@marko/build/dist/index.js
index 496e69e..5184fbb 100644
--- a/node_modules/@marko/build/dist/index.js
+++ b/node_modules/@marko/build/dist/index.js
@@ -210,7 +210,7 @@ const configBuilder = exports.configBuilder = ({
       test: /\.styl$/,
       use: styleLoaders(["stylus-loader", ensurePkgs(APP_DIR, ["stylus-loader", "stylus"])])
     }, {
-      test: file => file && !/\.([cm]?js|json|css|less|s[ac]ss|styl|wasm|marko)$/.test(file),
+      test: file => file && !/\.([cm]?js|json|css|less|s[ac]ss|styl|wasm|marko|md)$/.test(file),
       use: [{
         loader: require.resolve("file-loader"),
         options: {
@@ -408,7 +408,7 @@ function ensurePkgs(dir, pkgs) {
 
       for (const pkg of pkgs) {
         if (!resolveFrom.silent(dir, `${pkg}/package.json`)) {
-          throw new Error(`To load ${path.extname(info.resource)} files you must first install the following packages: ${pkgs.join(", ")}.`);
+          console.warn(`To load ${path.extname(info.resource)} files you must first install the following packages: ${pkgs.join(", ")}.`);
         }
       }
     }
diff --git a/node_modules/@marko/build/dist/util.js b/node_modules/@marko/build/dist/util.js
index 9cb7295..f845f45 100644
--- a/node_modules/@marko/build/dist/util.js
+++ b/node_modules/@marko/build/dist/util.js
@@ -30,13 +30,15 @@ const useAppModuleOrFallback = (dir, moduleName) => {
 
 const getDirectoryLookup = async (cwd, ignore) => {
   const lookup = {};
-  const filenames = await fastGlob(["**/*.marko"], {
+  // Added md here
+  const filenames = await fastGlob(["**/*.{marko,md}"], {
     cwd,
     ignore
   });
 
   for (const filename of filenames) {
-    lookup[fileNameToURL(filename).replace(/(^|\/)(?:index\/(?=index\.marko$))?([^/]+)?\.marko$/, "$1$2")] = path.join(cwd, filename);
+    // Added md here
+    lookup[fileNameToURL(filename).replace(/(^|\/)(?:index\/(?=index\.(?:marko|md)$))?([^/]+)?\.(?:marko|md)$/, "$1$2")] = path.join(cwd, filename);
   }
 
   return lookup;
@@ -67,10 +69,10 @@ const getRouterCode = async (cwd, ignore, production) => {
   } // If we're in watch mode (not production),
   // When new .marko files are added, we want to recompute the router code
 
-
+  // added md here
   const watchContext = production ? "" : `
     (_ => import(
-      /* webpackInclude: /\\.marko$/ */
+      /* webpackInclude: /\\.(marko|md)$/ */
       /* webpackExclude: /node_modules|components|build/ */
       /* webpackMode: "weak" */
       ${JSON.stringify(fileNameToURL(cwd + path.sep))} + _
